src/main/java/org/apache/ibatis/type/UnknownTypeHandler.java

```diff
@@ -22,20 +22,41 @@
 import java.sql.SQLException;
 import java.util.HashMap;
 import java.util.Map;
+import java.util.function.Supplier;
 
 import org.apache.ibatis.io.Resources;
+import org.apache.ibatis.session.Configuration;
 
 /**
  * @author Clinton Begin
  */
 public class UnknownTypeHandler extends BaseTypeHandler<Object> {
 
   private static final ObjectTypeHandler OBJECT_TYPE_HANDLER = new ObjectTypeHandler();
+  private final Configuration configuration;
```

Comment:
There is `configuration` field in the parent `BaseTypeHandler`, so it might be better to use a different name.

Reply:
@harawata Thanks for your review and comment! I've fixed via c524574855477d54f69858311ea20b2ac98d9803. 
And I've add TODO comment for rename to 'configuration' from 'config' after removing the deprecated property from parent class.